Solutions/JuniperIDP/Data Connectors/Connector_LogAnalytics_agent_JuniperIDP.json

```diff
@@ -0,0 +1,165 @@
+            { 
+                "instructions": [ 
+                    { 
+                    "parameters": { 
+                    "title": "Choose where to install the Windows agent:", 
+        "instructionSteps": [ 
+                { 
+                "title": "Install agent on Azure Windows Virtual Machine", 
+                "description": "Select the machine to install the agent on and then click **Connect**.", 
+                "instructions": [ 
+                    { 
+                    "parameters": { 
+                    "linkType": "InstallAgentOnVirtualMachine"
+                    }, 
+                    "type": "InstallAgent" 
+                    }
+                ]    
+            }, 
+                { 
+                "title": "Install agent on a non-Azure Windows Machine", 
+                "description": "Download the agent on the relevant machine and follow the instructions.", 
+                "instructions": [ 
+                    { 
+                    "parameters": { 
+                    "linkType": "InstallAgentOnNonAzure"  
+                    }, 
+                    "type": "InstallAgent" 
+                    } 
+                ] 
+            } 
+        ] 
+            }, 
+                "type": "InstructionStepsGroup" 
+                } 
+            ] 
+                }, 
+    { 
+            "title": "2. Configure the logs to be collected", 
+            "description":"Follow the configuration steps below to get Juniper IDP logs into Azure Sentinel. This configuration enriches events generated by Juniper IDP module to provide visibility on log source information for Juniper IDP logs. Refer to the [Azure Monitor Documentation](https://docs.microsoft.com/azure/azure-monitor/agents/data-sources-json) for more details on these steps.\n1. Download config file [juniper_idp.conf](link to the file).\n2. Login to the server where you have installed Azure Log Analytics agent.\n3. Copy juniper_idp.conf to the /etc/opt/microsoft/omsagent/**workspace_id**/conf/omsagent.d/ folder.\n4. Edit juniper_idp.conf as follows:\n\n\t i. change the listen port for receiving logs based on your configuration (line 3)\n\n\t ii. replace **workspace_id** with real value of your Workspace ID (lines 58,59,60,63)\n5. Save changes and restart the Azure Log Analytics agent for Linux service with the following command:\n\t\tsudo /opt/microsoft/omsagent/bin/service_control restart\n6. To configure a remote syslog destination, please reference the [SRX Getting Started - Configure System Logging](https://kb.juniper.net/InfoCenter/index?page=content&id=kb16502).",
+            "instructions":[
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "WorkspaceId"
+                        ],
+                        "label": "Workspace ID"
+                    },
+                    "type": "CopyableLabel"
+                }
+            ]
+    }
+]

```

Comment:
Please add a Metadata object within this file. See https://raw.githubusercontent.com/Azure/Azure-Sentinel/29c401e232cc7ce1ee91e7583133d3aa53e7e4af/DataConnectors/Templates/Connector_REST_API_template.json for further detail.